diff --git a/src/config.rs b/src/config.rs
index f0bed69..6ed6c32 100644
--- a/src/config.rs
+++ b/src/config.rs
@@ -25,8 +25,11 @@ pub struct OutputConfig {
 }
 
 #[derive(Deserialize, Debug, Clone)]
-pub struct AppearanceConfig {
+pub struct WaybarConfig {
     pub tag_icons: Option<Vec<String>>,
+    pub focused_style: Option<String>,
+    pub occupied_style: Option<String>,
+    pub empty_style: Option<String>,
 }
 
 // 1. 对应 [input.keyboard] 部分
@@ -67,7 +70,7 @@ pub enum KeyBindingEntry {
 pub struct Config {
     pub input: Option<InputConfig>,
     pub keybindings: Option<HashMap<String, KeyBindingEntry>>,
-    pub appearance: Option<AppearanceConfig>,
+    pub waybar: Option<WaybarConfig>,
     pub output: Option<HashMap<String, OutputConfig>>,
 }
 
@@ -103,7 +106,7 @@ impl Config {
         Config {
             input: None,
             keybindings: None,
-            appearance: None,
+            waybar: None,
             output: None,
         }
     }
